app/controllers/api/v1/migrate_users_to_account_api_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Api::V1::MigrateUsersToAccountApiController < Doorkeeper::ApplicationController
+  PAGE_SIZE = 100
+
+  before_action -> { doorkeeper_authorize! :migrate_users }
+
+  respond_to :json
+
+  rescue_from ActionController::ParameterMissing do
+    head :bad_request
+  end
+
+  def process_batch
+    page = params.fetch(:page).to_i
+
+    batch = User.page(page).per(PAGE_SIZE)
+
+    render json: {
+      users: migrate_batch_of_users(batch),
+      is_last_page: batch.last_page?,
+    }
+  end
+
+private
+
+  def migrate_batch_of_users(batch)
+    batch.map do |user|
```

Comment:
Is it worth having any kind of `next if subscription.migrated_to_account_api == true`?
I'm wondering what happens if the rake task flairs out half way through and we need to resume?

Reply:
The task in account-api doesn't change data which is already present in its database, so at worst we'll waste some time fetch a few pages of users which don't import.